pkg/setup/setup.go

```diff
@@ -292,7 +292,11 @@ func (s *Setup) installDefaultPlugins(httpClient *httpclient.Client) error {
 	if errCore != nil {
 		return errCore
 	}
-	return errEnterprise
+
+	if errEnterprise != nil {
+		s.logger.Error(fmt.Sprintf("cluster setup was successful but dcos-enterprise-cli could not be installed: %s", errEnterprise))
```

Comment:
Could the line be simplified using `s.logger.Errorf(format string, args ...interface{})` instead?